x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.ts

```diff
@@ -22,7 +23,15 @@ export function getOpsStatsCollector(
   metrics$: Observable<OpsMetrics>
 ) {
   let lastMetrics: MonitoringOpsMetrics | null = null;
-  metrics$.subscribe(metrics => {
+  metrics$.subscribe(_metrics => {
+    const metrics: any = cloneDeep(_metrics);
+    // Ensure we only include the same data that Metricbeat collection would get
+    metrics.process = omit(metrics.process, 'pid');
```

Comment:
nit: Any reason we can't just do `delete metrics.process.pid`? Seems this [method](https://lodash.com/docs/#omit) "creates" the object over again

Reply:
Done